app/javascript/mastodon/features/compose/index.jsx

```diff
@@ -86,8 +86,6 @@ class Compose extends PureComponent {
         onConfirm: () => logOut(),
       },
     }));
-
-    return false;
```

Comment:
`e.preventDefault()`; and `e.stopPropagation();` is doing the job of the return. the false is dead code.